project/mkdocs.yml.jinja

```diff
@@ -40,7 +40,8 @@ markdown_extensions:
 - pymdownx.tabbed
 - pymdownx.tasklist
 - toc:
-    permalink: "¤"
+    permalink: True
+    separator: "="
```

Comment:
Why an equal sign? The default `-` is more standard for URLs I believe.

Reply:
i thought previous sign look like equal sign.

if the sign is not important i can remove it completely

Reply:
Oh, the separator value has another meaning (it's the character replacing spaces in the anchor). The current symbol (¤) is a distinctive sign of mine (kinda like my personal brand). I'm OK to change it to a more generic symbol (typically # or §) as I don't use this template anymore.

Reply:
then it can be completely removed and let user change to their own after project generated

do note that your symbol may raise error as stated before here [^1]

[^1]: https://github.com/pawamoy/copier-poetry/pull/19#user-content-fn-3-7ca6a9bcc78651a41bc2bbf4d05739b2